typespec-tests/tsp/enum.tsp

```diff
@@ -45,6 +45,9 @@ model Operation {
   age: 50;
   priority: Priority;
   color: ColorModel;
+  priorityValue: Priority.Low;
```

Comment:
What does it mean? define the default value of `priorityValue` as `Priority.Low`?

Reply:
yes